patchelf.1

```diff
@@ -53,6 +53,9 @@ For instance, if an executable references one library libfoo.so, has
 an RPATH "/lib:/usr/lib:/foo/lib", and libfoo.so can only be found
 in /foo/lib, then the new RPATH will be "/foo/lib".
 
+.IP --shrink-prefix
+Work with --shrink-rpath, used in stage of build before installed to real dest path.
```

Comment:
Thanks for response

I use patchelf in a stage of rpmbuild, which install software files in a buildroot dir ( for example /home/rpmbuild/buildroot but which is not the root dir software actually run at ).
And in that case, patchelf would not find dynamic libs correctly. 

.IP --shrink-prefix PREFIX
Work with --shrink-rpath, search all directory in RPATH with prefix $PREFIX, insteal search all directory IN RPATH

For instance as above,  --shrink-prefix /root/build/,  /root/build/lib:/root/build/usr/lib:/root/build/foo/lib would be searched